site/en/blog/help-css-nesting-again/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: "layouts/blog-post.njk"
+title: Help choose the syntax for CSS Nesting
+authors:
+  - bramus
+  - argyle
+description: >
+  The CSS Working Group is continuing a debate over the best way to define nesting in CSS. And if you are someone who writes CSS, we’d like your help.
+date: 2022-12-16
+tags:
+  - css
+hero: image/vS06HQ1YTsbMKSFTIPl2iogUQP73/ZrMGmOcY6AH5J2tAblyO.jpg
+alt: >
+  A bird nest with a couple of blue eggs inside, surrounded by lush greenery.
+---
+
+After we ran [the previous survey to help choose the syntax for CSS Nesting](/blog/help-css-nesting), the CSS Working Group continued debate over the best way to define nesting in CSS. During the discussions new syntax ideas were suggested. To help the Working Group choose between any of these syntaxes, a new survey is launched.
+
+## The Options
+
+Based on [the results of the previous survey](/blog/help-css-nesting-survey), options 1 and 2 are longer considered. Instead two new options–numbered 4 and 5–were added.
```

Comment:
```suggestion
Based on [the results of the previous survey](/blog/help-css-nesting-survey), options one and two are longer being considered. Two new options (four and five) have been added.
```

Reply:
the `/blog/help-css-nesting-survey` link here is a 404 and I believe it should be `/blog/help-css-nesting-results/`

Reply:
Good catch @AaronForinton. Fixed in a later commit.